rwgame/MenuSystem.hpp

```diff
@@ -71,18 +71,14 @@ class Menu {
         }
     };
 
-    Menu(std::vector<MenuEntry> initial, int font = MenuDefaults::kFont,
+    Menu(std::initializer_list<MenuEntry>&& initial,
+         const glm::vec2& offset = glm::vec2(), int font = MenuDefaults::kFont,
          float size = 30.f)
-        : activeEntry(-1), font(font), size(size), entries(std::move(initial)) {
-    }
-
-    /**
-     * @brief creates a menu from the given menu items
```

Comment:
Please consider adding some kind of comment back.

Reply:
Earlier to was wrapper to create ptr of `Menu` class, for optional container, plain constructor is enough.